### PR TITLE
Basic api calls

### PR DESCRIPTION
**Pull request by AIn0n**

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "URL": "http://api.nbp.pl/api",
```
Comment (Panzer0) [inline src/config.json:2]: Is there a reason to use http here rather than https?

File: src/config.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "start_date": "2002-01-02",
+  "headers": {
+    "Accept": "application/json"
+  }
+}
```
Comment (Panzer0) [inline src/config.json:7]: It is good practice to finish files with a newline.

File: src/functions.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from datetime import date
+import json
+
+
+def date_to_iso8601(input_date: date) -> str:
```
Comment (Panzer0) [inline src/functions.py:5]: Wouldn't it be more readable to call the isoformat() method directly instead of defining a new one?

**Review by Panzer0 — APPROVED** [review]
Seems about right to me.

File: src/functions.py (hunk 3 above)
Comment (AIn0n) [reply src/functions.py:5]: It was done with TDD methods, so at the beginning I written function and test, after that I added functionality. And speaking strictly about readability - in this version we know exactly which time format we want to get.

**Pull request merged**